buildbot.mariadb.org/util/nginx.conf

```diff
@@ -17,7 +17,6 @@ server {
 	error_log  /var/log/nginx/buildbot.error.log;
 
 	# SSL configuration
-	ssl on;
```

Comment:
We are using the default in Ubuntu Bionic repos, which is version 1.14, removing it is premature.

Reply:
You can run with more versions, I'm running with 1.10 and 1.15 no problems.